### PR TITLE
6.9.z picks relating to config changes

### PR DESCRIPTION
**Pull request by JacobCallahan**
Included commits

- 024d2df1b Isolate rhel support, params, and deploy
- 93b9d0129 Added Metadata for RHEL and Satellite Version and Snap
- 0bd35f377 Added the satellite repository and rhel major version in repos.yml (#9255)
- 467a8cad1 keytab pulled differently (#9214)
- 9184dc0a0 add ssh tests for template sync
- 5dda5417a removing osp hostname from osp.yaml.template (#9263)
- 2c5e4710e Initial RemoteDB config and foreman-maintain CLI's (#9224)
- ddce5d07c Metadata for RHEL version (#9212)
- 8cba71e33 Add inital support for RHEL9
- 7ab373c40 Multiple RHEV test fixes (#9299)
- 95434f5e3 add token for github api calls
- 298eea48a expand content_host.yml to support custom deploy workflows per nick (#9327)
- b6d7499bf Add satutils and satclient to repos.yaml
- 643113883 refactor host tests to use rhel_contenthost fixture
- 9dcb4a662 rex cli refactor

### CODE REVIEW

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_activationkey.py tests/foreman/cli/test_activationkey.py tests/foreman/ui/test_activationkey.py

File: conf/content_host.yaml.template [hunk 1]
```diff
@@ -3,6 +3,13 @@ content_host:
         default:
         rhel9: # example of workflow override for version
     default_rhel_version: 7
+    rhel_versions:
+      6: {}
+      7: {}
+      8: {}
+      9_compose:
+        deploy_workflow: deploy-rhel-cmp-template
```
Comment (mshriver) [inline conf/content_host.yaml.template:11]: With `conf/supportability.yaml` this block shouldn't be necessary

File: conf/robottelo.yaml.template [hunk 2]
```diff
@@ -17,4 +17,6 @@ ROBOTTELO:
   RUN_ONE_DATAPOINT: false
   # Satellite version supported by this branch
   # UNDR version is used for some URL composition
-  SATELLITE_VERSION: "6.9"
+  SATELLITE_VERSION: "7.0"
```
Comment (mshriver) [inline conf/robottelo.yaml.template:20]: Version needs to be 6.9 here

File: conf/supportability.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 supportability:
   content_hosts:
     rhel:
-      versions: [6, 7, 8, 9]
+      versions: [6, 7,'7_fips', 8, '8_fips', 9]
```
Comment (mshriver) [inline conf/supportability.yaml:4]: no rhel9 support on 6.9.z

File: tests/foreman/cli/test_remoteexecution.py [hunk 4]
```diff
@@ -105,8 +104,8 @@ def test_positive_run_default_job_template_by_ip(self, fixture_vmsetup):
     @pytest.mark.tier3
     @pytest.mark.pit_client
     @pytest.mark.pit_server
-    @pytest.mark.parametrize('fixture_vmsetup', [{'nick': 'rhel7'}], ids=['rhel7'], indirect=True)
-    def test_positive_run_job_effective_user_by_ip(self, fixture_vmsetup):
+    @pytest.mark.rhel_ver_list([7, 8, 9])
```
Comment (mshriver) [inline tests/foreman/cli/test_remoteexecution.py:107]: No rhel 9 support, though without it in `supportability.yaml` having it here shouldn't actually do anything.

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_activationkey.py tests/foreman/cli/test_activationkey.py tests/foreman/ui/test_activationkey.py

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_activationkey.py tests/foreman/cli/test_activationkey.py tests/foreman/ui/test_activationkey.py

**Comment by JacobCallahan** [comment]
ok, results from the PRT run previous to this comment. API and CLI tests ran perfectly. It had issues with UI tests, so a backport has been created there for this as well.
https://github.com/SatelliteQE/airgun/pull/677
Once that is merged, I will run just the UI tests again. If those go well, I think we're well within a margin of error for a 6.9.z automation run.

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_activationkey.py
airgun: 677

**Review by mshriver — APPROVED** [review]

**Pull request merged**